README.md

```diff
@@ -4,6 +4,17 @@
 
 Install [Nextflow](https://www.nextflow.io/): `curl -s https://get.nextflow.io | bash`
 
+This command will create a `nextflow` executable in the current directory. To simplify usage, consider moving this executable to a directory that is available on `$PATH`. One common place for this is a `bin/` directory in your home folder:
+
+``` bash
+mkdir -p ~/bin                                      # Creates a bin directory in the home folder
+mv nextflow ~/bin                                   # Moves nextflow to that directory
+echo "export PATH=$HOME/bin:\$PATH" >> ~/.bashrc    # Make the directory accessible on $PATH
```

Comment:
Prefer this, to avoid the need for backslash and to handle any PATH components that have spaces:

```bash
echo 'export PATH="$HOME/bin:$PATH"' >> ~/.bashrc
```